SP-2525: Add comparisons between collections to plot-navigator

ctslater opened the pull request:
The main feature is the "Compare" page, which you can get to by selecting a specific plot type then using the "Compare Collections" button. Deployed on usdf-rsp-dev for testing.

Also includes:
* Some infrastructure to support testing against local data, without needing a butler repo.
* The "More Plots" page
* A few syntax and UI cleanups.

This does not fix the bug where downselecting by band doesn't change the lightbox next/forward buttons. Still need to do that in a new PR.

ctslater committed:
Add local testing data and routes.

ctslater committed:
Make lightbox contents bigger.

ctslater committed:
Add comparison between collections.

ctslater committed:
Sort collection names in CompareCollectionButton.

ctslater committed:
Update versions.

ctslater committed:
Code reformatting.

ctslater committed:
Fix syntax.

ctslater committed:
Add lower prev/next buttons to comparison page.

ctslater committed:
Display plots via direct urls from summary file.

ctslater committed:
Add More Plots page.

ctslater committed:
Add collection note display. Sort more plots.

ctslater committed:
Fix "Back to collections" link.

jrmullaney commented on app/plot/[repo]/[collection]/[plotName]/compare/[repo2]/[collection2]/page.js:
```diff
@@ -0,0 +1,104 @@
+
+    const plotDisplays2 = plotEntries2.map((entry, n) => 
+        ({dataId: JSON.parse(entry.dataId), plot: <PlotDisplay key={n} showPermalink={false} plotEntry={ ({...entry, repo: repo2,
+        permalink: `/plot/${encodeURIComponent(repo)}/${encodeURIComponent(collection)}/${encodeURIComponent(plotName)}/${encodeDataId(entry.dataId)}`}) } />})
```
Should this definitely be `repo` and not `repo2` in `encodeURIComponent(repo)`?

jrmullaney commented on app/plot/[repo]/[collection]/[plotName]/compare/[repo2]/[collection2]/page.js:
```diff
@@ -0,0 +1,104 @@
+        const bandsOrder = ['u', 'g', 'r', 'i', 'z', 'y']
+
+        if('tract' in dataIdA && 'tract' in dataIdB) {
+            const tract_comparision = dataIdA.tract - dataIdB.tract
```
Is `tract_comparision` a typo for `tract_comparison`? i.e., without the second "i".

jrmullaney reviewed (approved):
Just a couple of things. Please see my comments on the JIRA ticket, too.

ctslater committed:
Fixes to plot comparison code.

- Make the band selector work.
- Display a message when there are no plots to show.
- Consistently use A/B instead of 1/2.

ctslater committed:
Clean up dataId sorting.
